src/explorer/explorerTreeController.ts

```diff
@@ -27,7 +27,7 @@ implements vscode.TreeDataProvider<vscode.TreeItem> {
     this._connectionController.addEventListener(
       DataServiceEventTypes.CONNECTIONS_DID_CHANGE,
       () => {
-        this.refresh();
+        void this.refresh();
```

Comment:
Isn't this now sync so no reason for void? Or maybe I'm misunderstanding and you're doing void when ignoring any return value and not just returned promises.

Reply:
Thx for catching it! Yes, I changed the function to sync after some iterations and this is a leftover. Removed void.